modules/bmo-editing-a-baremetalhost-resource.adoc

```diff
@@ -0,0 +1,44 @@
+// This module is included in the following assemblies: 
+//
+// post_installation_configuration/bare-metal-configuration.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="editing-a-baremetalhost-resource_{context}"]
+= Editing a BareMetalHost resource
+
+After deploying a cluster, you might need to edit a node's `BareMetalHost` resource. For example, if you deploy a cluster with the {ai-full} and you need to add the baseboard management controller (BMC) host name or IP address, or you want to move a node from one cluster to another without deprovisioning it, you must edit the node's `BareMetalHost` resource. To edit a node's `BareMetalHost` resource without deprovisioning the node, use the following procedure.
```

Comment:
Carrying the BMC hostname discussion here: Would that not be the domain name like google.com (and is that not different from what hostname means?)

Reply:
Makes sense, thanks for the explanation!